From: Jelmer Vernoo캐 <jelmer@debian.org>
Subject: Automatically generated patch (0.18.0-1)

Last (up to) 3 git changes, FYI:

commit 4d7c6c481bd37f3d03e3d538fbe336fb5f5b3781
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Mon Jul 31 01:28:32 2017 +0000

    releasing package dulwich version 0.18.0-1

commit 44ae927d4054a2969d145a6e0fc35ebe4e958044
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Mon Jul 31 01:23:23 2017 +0000

    Drop auto patch.

commit ca2d0d96a567276381b5f7afd9777af547cd8e5a
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Mon Jul 31 01:22:33 2017 +0000

    New upstream release.
---

--- dulwich-0.18.0.orig/dulwich.egg-info/SOURCES.txt
+++ dulwich-0.18.0/dulwich.egg-info/SOURCES.txt
@@ -6,7 +6,6 @@ COPYING
 MANIFEST.in
 Makefile
 NEWS
-PKG-INFO
 README.md
 README.swift.md
 TODO
@@ -18,25 +17,6 @@ tox.ini
 bin/dul-receive-pack
 bin/dul-upload-pack
 bin/dulwich
-debian/.gitignore
-debian/changelog
-debian/compat
-debian/control
-debian/copyright
-debian/gbp.conf
-debian/pypy-dulwich.lintian-overrides
-debian/python-dulwich.lintian-overrides
-debian/python3-dulwich.lintian-overrides
-debian/rules
-debian/watch
-debian/patches/series
-debian/patches/sources_debian
-debian/source/format
-debian/source/options
-debian/tests/control
-debian/tests/testsuite2
-debian/tests/testsuite3
-debian/upstream/signing-key.asc
 docs/Makefile
 docs/conf.py
 docs/index.txt
